cmd/run.go

```diff
@@ -58,9 +62,12 @@ func init() {
 	runCmd.Flags().StringVarP(&sshKeyPath, "ssh-key-path", "i", "", "path to private ssh key")
 	runCmd.Flags().UintVarP(&sshTimeout, "ssh-connect-timeout", "c", 30, "time to wait (in secs) for hosts to accept connection")
 	runCmd.Flags().UintVarP(&timeout, "timeout", "t", 300, "total time to wait (in secs) for host processing to complete")
+	runCmd.Flags().StringSliceVarP(&certIPs, "cert-ips", "", []string{}, "list of ips in ca cert (comma-delimited)")
+	runCmd.Flags().StringSliceVarP(&certDNSNames, "cert-dns-names", "", []string{}, "list of dns names in ca cert (comma-delimited)")
 
 	runCmd.MarkFlagRequired("node-ips")
 	runCmd.MarkFlagRequired("ssh-key-path")
+	runCmd.MarkFlagRequired("cert-dns-names")
```

Comment:
Why not add a default value like `rancher.example.org`?

Reply:
Just hadn't gotten to it, but I've since added.